packages/schema/lib/functional-constraints/AuthFieldisSafe.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const jsonschema = require('jsonschema');
+
+const AUTH_FIELD_ID = '/AuthFieldSchema';
+const AUTH_FIELDS_ID = '/AuthFieldsSchema';
+
+const FORBIDDEN_KEYS = [
+  'access_token',
+  'refresh_token',
+  'api_key',
+  'password',
+  'secret',
+];
+
+const checkAuthField = (field) => {
+  const errors = [];
+
+  if (FORBIDDEN_KEYS.includes(field.key) && field.isSafe === true) {
+    errors.push(
+      new jsonschema.ValidationError(
+        `Cannot set isSafe=true for the sensitive key "${field.key}".`,
```

Comment:
```suggestion
        `cannot set isSafe as true for the sensitive field with key "${field.key}".`,
```

Starting with a lowercase because the `err.stack` is formed by prepending the `err.property` to this message to form a sentence.